src/components/Dashboard/Dashboard.tsx

```diff
@@ -0,0 +1,193 @@
+import { Loader2 } from 'lucide-react';
+import React, { ReactElement, useState, useEffect } from 'react';
+import WidgetHeader from '@/components/WidgetHeader/WidgetHeader';
+import WidgetContainer from '@/components/WidgetContainer/WidgetContainer';
+import useFetch from '@/utils/fetchData';
+import Widget from '@/components/Widget/Widget';
+
+// Define the types for the component props
+interface DashboardContainerProps {
+	children: React.ReactNode;
+}
+
+interface DashboardSectionProps {
+	title: string;
+	children: React.ReactNode;
+}
+
+interface WidgetProps {
+	category: string;
+	[key: string]: any;
+}
+
+interface DahsboardProps {
```

Comment:
Please correct spelling to `DashboardProps`

Reply:
done